### PR TITLE
feat: MongoDB & Prisma [serverless]

### PR DESCRIPTION
**Pull request by EmperorOrokuSaki**

### CODE REVIEW

File: serverless/prisma/tsconfig.json
Comment (zoruka) [inline serverless/prisma/tsconfig.json:1]: I think we might add this file in the root `.eslintrc.js` file as a project to recognize correctly the prisma project

Comment (zoruka) [inline serverless/prisma/tsconfig.json:1]: Do we need to keep all the comments here?

File: serverless/prisma/client/index.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+main()
+  .then(async () => {
+    await prisma.$disconnect()
+  })
+  .catch(async (e) => {
+    console.error(e)
+    await prisma.$disconnect()
+    process.exit(1)
+  })
```
Comment (zoruka) [inline serverless/prisma/client/index.ts:32]:
```suggestion
main()
  .catch(async (e) => {
    console.error(e)
  })
  .finally(async () => {
    await prisma.$disconnect();
  });
```

File: serverless/prisma/client/index.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+async function main() {
```
Comment (jsonsivar) [inline serverless/prisma/client/index.ts:5]: I don't know if this lib needs a "main" function.  We should create a function called `getAllTokens` or similar and put this logic there.

**Review by jsonsivar — DISMISSED** [review]
Hey for this I think we should make prisma a lib in the serverless package and not create another sub-package? Ie I dont think prisma folder needs its own package.json or tsconfig.json.

File: serverless/prisma/client/index.ts (hunk 2 above)
Comment (zoruka) [reply serverless/prisma/client/index.ts:5]: maybe just leave a comment saying that is a example is enough

**Review by zoruka — APPROVED** [review]

**Pull request merged**